substrate/frame/support/src/traits/tokens/fungible/freeze.rs

```diff
@@ -15,7 +15,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! # Freeze
```

Comment:
This is indeed the right place to document this. Any methods that can also be improved?